src/xbot/common/properties/PropertyManager.java

```diff
@@ -102,17 +103,48 @@ public void saveOutAllProperties() {
         permanentStore.saveToDisk();
     }
     
+    /**
+     * @deprecated You should use createProperty(String key, Boolean defaultValue, PropertyPersistenceType persistenceType) 
```

Comment:
Instead of deprecating these factory functions maybe it would be friendlier to have like createProperty and createEphemeralProperty or something so it's a little leaner?